dwave/optimization/_model.pyx

```diff
@@ -140,6 +140,48 @@ cdef class _Graph:
         """
         return sum(sym.state_size() for sym in self.iter_decisions())
 
+    def input(
+        self,
+        lower_bound: float = -float("inf"),
+        upper_bound: float = float("inf"),
+        integral: bool = False,
```

Comment:
The Python-style typing doesn't get used in the `.pyx`, so should be removed for consistency.

However I would recommend using Cython typing (e.g. `double upper_bound = float("inf")`) for consistency with `Input`'s constructor.